proposals/46-store-my-text-example-app.md

```diff
@@ -0,0 +1,111 @@
+# Store-my-text Filecoin example app
+
+Authors: @zenground0 @rvagg
+
+Initial PR: https://github.com/protocol/web3-dev-team/pull/46
+
+## Purpose &amp; impact 
+
+#### Background &amp; intent
+
+_Describe the desired state of the world after this project? Why does that matter?_
+
+We have an extremely simple example application, _and_ a functioning service that has real-world utility that exercises Filecoin and deal-making.
+
+ * A **very** simple web application, minimal UI fluff and distraction: **Save my text to Filecoin**
+   - Text field to enter some UTF-8 text
+   - Field to submit your email address
+   - A button to "Submit"
+ * User enters some non-empty text and their email address and presses "Submit"
+ * Application performs a simple confirmation-of-email ("click this link") to confirm a real user and manage trivial DoS
+ * Application handles storage of text on Filecoin and sends confirmation email
+   - Application mediates the storage deal-making process for small content - finding miners, batching content, etc.
+ * CID provides persistent "proof-of-text" to the user
+
+Initially this is just a very simple system that demonstrates (and exercises, and educates us on) the full deal flow for simple content.
+
+There's plenty of extensions that could be made to this, turning it into a genuinely useful application, such as:
+
+ * IPFS integration
+ * Other input types (images, etc. getting toward the NFT use-case)
+ * Payments ("Deposit X FIL to this address...", "Enter your CC to purchase X FIL")
```

Comment:
This one is important to do (does not have to be in rev 0).  It's going to reveal a whole other set of blockchain pain points that developers will face, and we should face them too, to see how we can improve.